docs/integrations/lightgbm.rst

```diff
@@ -3,33 +3,74 @@
 Neptune-LightGBM Integration
 ============================
 
+|Run on Colab|
+
+What will you get with this integration?
+----------------------------------------
+
+|lightgbm-tour|
+
 |lightGBM| is a popular gradient boosting library. The integration with Neptune lets you log training and evaluation metrics and have them visualized in Neptune.
```

Comment:
Could you make the things you will get into bullets?

I'd also add the tip like here https://docs.neptune.ai/integrations/pytorch.html